app/models/crop.rb

```diff
@@ -354,6 +335,23 @@ def self.search(query)
     end
   end
 
+  private
+
+  def add_names_to_list(names_to_add, list_name, col_name)
+    cropbot = Member.find_by_login_name('cropbot')
+    names_to_add.each do |n|
+      if self.send("#{list_name}_names").exists?(["#{col_name} LIKE ?", n])
```

Comment:
Not sure you can avoid that without creating a view or similar (```SELECT foo as bar FROM blah```) - that quickly is going to upset activerecord for an edge case.

You *might* be able to use ```alias_attribute :name :scientific_name``` and swap it for ```.where(name: ...)```